packages/runner-ct/src/app/app.scss

```
@@ -50,7 +50,8 @@ main.app-ct {
     align-items: center;
     justify-content: center;
 
-    a {
+    a#menu-toggle { 
+      width: 20px;
```

maybe a bigger width?
```suggestion
    a#menu-toggle { 
```